shared/src/main/scala/coop/rchain/shared/LongOps.scala

```diff
@@ -0,0 +1,20 @@
+package coop.rchain.shared
+import java.lang.Long.numberOfLeadingZeros
+import java.util.Locale
+
+object LongOps {
+
+  /**
+    * Formats size in bytes to human-readable format
+    * Avoids use of slow math operations and loops
+    *
+    * Assumes 1 KB = 1024 bytes, not 1000 bytes as some other methods do
+    */
+  def toHumanReadableSize(value: Long): String =
```

Comment:
My take on the `<name>Ops` objects is that they should contain [implicit classes](https://docs.scala-lang.org/overviews/core/implicit-classes.html) that enrich existing types.  

So `LongOps` here would contain:

```scala
  implicit class RichLong(value: Long) {

    /**
      * Formats size in bytes to human-readable format
      * Avoids use of slow math operations and loops
      *
      * Assumes 1 KB = 1024 bytes, not 1000 bytes as some other methods do
      */
    def toHumanReadableSize: String =
      if (value < 1024) {
        value + " B"
      } else {
        val z = 63 - numberOfLeadingZeros(value)
        "%.1f %sB".formatLocal(Locale.US, value.toDouble / (1L << z), " KMGTPE".charAt(z / 10))
      }
  }
```

When `LongOps` is brought into scope, you could then use it like:

```scala
  def showStoreUsage(storeUsage: StoreUsage): String =
    s"""Store metrics:
       | - Size On Disk: ${storeUsage.sizeOnDisk.toHumanReadableSize}
       | - Data Entries: ${storeUsage.dataEntries}
       |""".stripMargin
```

Reply:
Thanks for a tip! Here's the place where all my theoretical knowledge from scala books lost somewhere when they're needed in practice.